### ISSUE TITLE
Integration with erb?

### ISSUE THREAD

**Issue by DanielMoreno58** [thekalinga/bootstrap4-vscode#44]
Hi, I'm a developer from ruby and I use erb for my templates instead html

**Is there a way I can integrate this tool with erb?**

If not, could this be a posibility for all the developers that we are using erb?
I appreciate every support.

**Comment by lastobelus** [comment]
edit package.json in the extension and add

```
			{
				"language": "erb",
				"path": "./snippets.json"
			}
```

to `"snippets"` section.

**Comment by cwy007** [comment]
https://vscode-docs.readthedocs.io/en/stable/extensions/install-extension/
![](https://ws4.sinaimg.cn/large/006tNbRwly1fx5pp2rkt2j31kw0n24ao.jpg)

**Comment by thekalinga** [comment]
Fixed in version 5.0.0. Please check & let me know if there are any issues
